app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -445,6 +445,15 @@ class LuciBuildService {
       log.fine('Skipping schedulePostsubmitBuilds as there are no targets to be scheduled by Cocoon');
       return;
     }
+
+    // Delay till GoB mirror is at commit, or we hit max acceptable delay.
+    for (int delaySecs in [0, 5, 10, 30]) {
```

Comment:
AppEngine requests are limited to 60 seconds, and this will time it out. Worst case, this adds 45 seconds for the polling, and IIRC the http requests add a 10 second timeout for each request.

I'm not sure this would work if we needed to support a mono-repo as well, as the mirror times could take a few minutes to sync flutter, engine, and dart